dev/tools/dartdoc.dart

```diff
@@ -90,6 +94,20 @@ dependencies:
   createIndexAndCleanup();
 }
 
+void createFooter(String footerPath) {
+  ProcessResult gitResult = Process.runSync('git', <String>['rev-parse', 'HEAD']);
+  if (gitResult.exitCode != 0)
+    exit(gitResult.exitCode);
```

Comment:
does a message get output by git in this case? maybe we should print something to the console here just to make it easier for us to figure out what went wrong